kubejs/server_scripts/base/recipes/apotheosis/boss_gear/twilight.js

```diff
@@ -0,0 +1,243 @@
+ServerEvents.highPriorityData((event) => {
```

Comment:
Alright. I had built it this way since it's overwriting the original data files, which are unfortunately under a `twilight` folder.

I can just build up a separate script to override all the original twilight entries, then move everything to twilight_forest

Reply:
If the originals are in a `twilight` folder then I get it, please disregard :)